File: GTTS/INL.TrainingService.Models/TrainingEventAttendance_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
    public class TrainingEventAttendance_Item : ITrainingEventAttendance_Item
    {
        public long? TrainingEventAttendanceID { get; set; }
        public long? TrainingEventRosterID { get; set; }
        public DateTime AttendanceDate { get; set; }
        public bool AttendanceIndicator { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public DateTime ModifiedDate { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/training-event-attendance_item.ts


import { ITrainingEventAttendance_Item } from './itraining-event-attendance_item';

export class TrainingEventAttendance_Item implements ITrainingEventAttendance_Item {
  
	public TrainingEventAttendanceID?: number;
	public TrainingEventRosterID?: number;
	public AttendanceDate: Date = new Date(0);
	public AttendanceIndicator: boolean = false;
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
  
}






